packages/fxa-auth-server/lib/routes/subscriptions/stripe-webhook.ts

```diff
@@ -149,6 +174,33 @@ export class StripeWebhookHandler extends StripeHandler {
     );
   }
 
+  /**
+   * Handle `invoice.created` events, if the subscription is for an invoice
+   * customer (PayPal), set auto_advance false and finalize.
+   */
+  async handleInvoiceCreatedEvent(request: AuthRequest, event: Stripe.Event) {
+    // Type-guard to require paypalHelper.
```

Comment:
Since we are using TypeScript and type guards are a thing in TS, it might be a little confusing to call this a type guard.

Reply:
A Type Guard in TS is pretty much anything that narrows the type, which this is doing.
https://basarat.gitbook.io/typescript/type-system/typeguard illustrates a variety of them, including one of this type.